text-summarization/run_service.py

```diff
@@ -10,39 +11,44 @@ def main():
     parser = argparse.ArgumentParser(prog="run-snet-services")
     parser.add_argument("--daemon-config-path", help="Path to daemon configuration file", required=False)
     args = parser.parse_args(sys.argv[1:])
-
-    def handle_signal(signum, frame):
-        snetd_p.send_signal(signum)
-        service_p.send_signal(signum)
-        snetd_p.wait()
-        service_p.wait()
-        exit(0)
-
-    signal.signal(signal.SIGTERM, handle_signal)
-    signal.signal(signal.SIGINT, handle_signal)
```

Comment:
Does this new system exit cleanly?

e.g. if I use ctrl-c after starting up `python run_service.py ...`, does it cleanly shut down all the spawned processes too?

Reply:
```
    1 ?        Ss     0:03 python3.6 run_service.py --daemon-config-path snetd.testnet.config.json
    8 ?        Sl     6:50 snetd serve --config snetd.testnet.config.json
    9 ?        S      0:00 python3.6 -m services.translate_server
```
If user kill any of these 3 processes, the others will be killed too (by `kill_and_exit()`).

Reply:
ok, as long as using ctrl-c kills everything. I think I previously had issues with the daemon and service hanging around after I ctrl-c'ed run_service.py which is why I was catching SIGINT.

Reply:
(Maybe it was only an issue on windows due to differences in how processes are spawned)